doc/Installation/Installation-CentOS-7-Apache.md

```diff
@@ -102,6 +102,9 @@ Install the policy tool for SELinux:
     setsebool -P httpd_can_sendmail=1
 
 ##### Allow fping
+
+    cd /opt/librenms
```

Comment:
Is this actually needed as we don't do anything within /opt/librenms and one of the next lot of commands moves you out of that dir anyway?

Reply:
You are right, sorry for the noise. Please ignore this PR.